packages/back-end/src/services/audit.ts

```diff
@@ -68,18 +68,25 @@ export async function getWatchedAudits(
     return [];
   }
 
+  const allWatchedThings = doc.experiments.concat(doc.features);
```

Comment:
I see what you mean, but by only having one query and sending docs to the frontend, it makes the frontend code a whole lot simpler. This is also only being used for the activity notifications anyways. Additionally, I know we wanted to make watching a lot more dynamic, so I wasn't sure if I should make different functions for each different query. I can still change it though if needed.

Reply:
We can still just have this single function return an array with both items in it.  We would run two separate mongo queries, then merge the arrays in code and return it, all from within this function.

I'm mostly worried about the performance of this database query and how it will scale as we add more items.  Doing separate smaller queries is probably going to work better.